### PR TITLE
Update GoogleServices.xml

### PR DESCRIPTION
**Pull request by hstseverywhere**
https://hstspreload.org/?domain=gmail.com
https://hstspreload.org/?domain=googlemail.com

### CODE REVIEW

**Comment by pipboy96** [comment]
Need to check if this is preloaded in Firefox.

**Comment by pipboy96** [comment]
Yes, appears to be preloaded.

File: src/chrome/content/rules/GoogleServices.xml [hunk 1]
```diff
@@ -34,23 +34,18 @@
 		- Waze.com.xml
 		- WebM_Project.org.xml
 		- With_Google.com.xml
+
+	HSTS:
+		- gmail.com
+		- googlemail.com
```
Comment (pipboy96) [inline src/chrome/content/rules/GoogleServices.xml]: TBH there's no real need in adding this.

File: src/chrome/content/rules/GoogleServices.xml [hunk 2]
```diff
@@ -34,23 +34,18 @@
...
-		<target host="www.googlemail.com" />
-	<securecookie host="^gmail\.com$" name=".+" />
-	<securecookie host="^www\.gmail\.com$" name=".+" />
-	<securecookie host="^googlemail\.com$" name=".+" />
-	<securecookie host="^www\.googlemail\.com$" name=".+" />
-
 
 	<rule from="^http:"
 			to="https:" />
+
```
Comment (pipboy96) [inline src/chrome/content/rules/GoogleServices.xml]: No need of extra line feed here.

**Comment by J0WI** [comment]
Closing for now due https://github.com/EFForg/https-everywhere/pull/17809#issuecomment-485389064

**Pull request closed without merging**